packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3052,6 +3055,15 @@ class _TraversalSortNode implements Comparable<_TraversalSortNode> {
 /// obtain a [SemanticsHandle]. This will create a [SemanticsOwner] if
 /// necessary.
 class SemanticsOwner extends ChangeNotifier {
+  /// Creates a [SemanticsOwner] manages zero or more [SemanticsNode] objects.
+  SemanticsOwner({
+    required this.onSemanticsUpdate,
+  });
+
+  /// Updates to the internal state of the [SemanticsOwner]'s [SemanticsNode]s
+  /// are accumulted into one [SemanticsUpdate] that can be delegated with the
+  /// onSemanticsUpdate callback.
```

Comment:
This doc is vague... I would talk about how this gets called during sendSemanticsUpdate, which creates a SemanticsUpdate that's passed to this thing. And its the expectation that this callback dispatches the updated to the FlutterView that is associated with this PipelineOwner/SemanticsOwner or something like that.